tests/units/actions/test_chain_of_actions_base.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from abc import ABCMeta
+
+from urnai.actions.chain_of_actions_base import ChainOfActionsBase
+
+
+class TestChainOfActionsBase(unittest.TestCase):
+
+	def test_abstract_methods(self):
+		ChainOfActionsBase.__abstractmethods__ = set()
+
+		class FakeChainOfActions(ChainOfActionsBase):
+			def __init__(self):
+				super().__init__()
```

Comment:
I think it would be better to place this class outside the `TestChainOfActionsBase`, so it's easier to reuse it for future tests if the need arises.

Reply:
Does this apply to other tests similar to this one, such as the test for ActionBase?